src/cowrie/ssh/factory.py

```diff
@@ -72,6 +75,11 @@ def startFactory(self):
             except IOError:
                 pass
 
+        try:
+            self.ourVersionString = CowrieConfig().get('ssh', 'version')
+        except NoOptionError:
```

Comment:
You can use the `fallback` option in get() here. that removes the try/except part.